app/assets/stylesheets/custom.scss

```diff
@@ -2,6 +2,8 @@
 @import 'bootstrap';
 @import 'define_color.scss';
 
+$gray-medium-light: #eaeaea;
```

Comment:
Please move this variable to the external file where you declare the color variable